cli/execute.py

```diff
@@ -52,6 +28,10 @@ def ansible_playbook(playbook, args, inventory="local_hosts"):
      :args: all arguments passed for the playbook
      :inventory: the inventory file to use, default: local_hosts
     """
+    verbosity = args["verbose"]
+    display = Display(verbosity=verbosity)
+    import __main__ as main
+    setattr(main, "display", display)
```

Comment:
That's how ansible 2 sets verbosity.
It's a hack, because ansible-playbook isn't fully supported to be consumed from python API instead of CLI.
I am looking for a better way. So far, that's all I've got